modules/nodes-scheduler-taints-tolerations-about.adoc

```diff
@@ -216,9 +244,23 @@ These tolerations ensure that the default pod behavior is to remain bound for fi
 
 You can configure these tolerations as needed. For example, if you have an application with a lot of local state, you might want to keep the pods bound to node for a longer time in the event of network partition, allowing for the partition to recover and avoiding pod eviction.
```

Comment:
How much is _a lot of state_?

Reply:
That is a general statement, an example of why you would use `tolerationSeconds`. Swiped from the upstream.